packages/invoices/src/OutboundShipment/api/api.ts

```diff
@@ -237,16 +237,30 @@ export const getOutboundQueries = (sdk: Sdk, storeId: string) => ({
     const input = {
       insertOutboundShipmentLines: draftStocktakeLines
         .filter(
-          ({ type, isCreated }) =>
-            isCreated && type === InvoiceLineNodeType.StockOut
+          ({ type, isCreated, numberOfPacks }) =>
+            isCreated &&
+            type === InvoiceLineNodeType.StockOut &&
+            numberOfPacks > 0
```

Comment:
Not the most elegant code! suggestions welcome 🤔

Reply:
uh, when testing an earlier PR and having problems.. I did exactly this! 😆